EmployeeTaskSolicy/Model/Employee.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace EmployeeTaskSolicy.Model
+{
+    public class Employee
+    {
+        [Required]
+        [Range(1, int.MaxValue, ErrorMessage = "Id must be positive number!")]
+        public int Id { get; set; }
+        [Required]
+        public string FirstName { get; set; }
+        [Required]
+        public string LastName { get; set; }
+        [Required]
+        [Range(16, 65, ErrorMessage = "Age must be more than {1} and less than {2}")]
```

Comment:
age range is [16-65], but in error message it says it should be [1-2]??

Reply:
No its correct. The Range works with that way its string format